src/site/content/en/react/code-splitting-suspense/index.md

```diff
@@ -95,11 +95,10 @@ weak network connections.
 
 To better demonstrate how this works:
 
-+  Mouse over the editor and press the **Show** button to preview the app.
-+  Open the DevTools by pressing `CMD + OPTION + i` / `CTRL + SHIFT + i`.
-+  Click on the **Network** tab.
-+  Click the [**Throttling**](https://developers.google.com/web/tools/chrome-devtools/network/#throttle) dropdown and select `Slow 3G`.
-+  Click the button on the application.
+{% Instruction 'preview' %}
+{% Instruction 'devtools-network' %}
+-  Click the [**Throttling**](https://developers.google.com/web/tools/chrome-devtools/network/#throttle) dropdown and select `Slow 3G`.
+-  Click the button on the application.
```

Comment:
```suggestion
-  Click the **Click Me** button in the application.
```
For clarity. (Despite DSG, I'm keeping "button" in this case because of the change in context--favoring clarity over brevity.)